docs/api-client/README.md

```diff
@@ -0,0 +1,929 @@
+# REST API
+
+The OpenLaw REST API in APIClient.js is an interface for querying, saving, and changing data in an OpenLaw instance. The API methods are categorized below.
+
+## Template
+
+### getTemplate
+
+Get template by its title.
+
+```
+GET /template/raw/:title
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `title` | `string` | **Required.** The title of the template. |
+
+Example
+
+```
```

Comment:
@adridadou what do you think is the best way to work in the examples? (Before merging this, I think we should probably either remove the example category or include them).

Reply:
Per discussion during the standup today, i'll include examples for everyone's review too.

Reply:
Sounds great, @Jdville03! Feel free to reach out if you need anything there.